src/main/java/org/magicdgs/http/jsr203/HttpFileSystem.java

```diff
@@ -83,7 +111,20 @@ public String getSeparator() {
 
     @Override
     public Path getPath(final String first, final String... more) {
-        throw new UnsupportedOperationException("Not implemented");
+        final String path = first + String.join(getSeparator(), more);
```

Comment:
Check no null for `first` (and maybe `more, depending on the contract of `String.join`

Reply:
Done for both.